.circleci/config.yml

```diff
@@ -35,6 +35,19 @@ commands:
   ###
   # App commands
   ###
+  app-save-env-file:
+    steps:
+      - run:
+          name: "App: save env file"
+          command: |
+            cd app 
+            echo "DOMAIN_NAME=$DOMAIN_NAME" >> .env
+            echo "CLIENT_ID=$CLIENT_ID" >> .env
+            echo "AUTH0_SCOPE=$AUTH0_SCOPE" >> .env
+            echo "AUDIENCE=$AUDIENCE" >> .env
+            echo "DEFAULT_URL=$DEFAULT_URL" >> .env
+            cd ..
+
```

Comment:
Are these the only environment variables needed? Do you need to add vblocks context somewhere in this config.yml file?

Reply:
These are the only ones needed for the App. We will later need to add vblocks context for the server, but that will be in `server/env`

Reply:
Now that I'm looking at this again, is this necessary. We're copying env vars to a file, which is the fallback when the env vars are not present. Is that the behavior of the module? That it falls back to the `.env` file for env vars that aren't present?